cms/templates/manage_users_lib.html

```diff
@@ -0,0 +1,181 @@
+                  <span class="admin-role notoggleforyou">${_("Promote another member to Admin to remove admin rights")}</span>
+                </li>
+              % endif
+            % elif is_staff:
+              <li class="action action-role">
+                <a href="#" class="make-instructor admin-role add-admin-role">${_("Add Admin Access")}</span></a>
+                <a href="#" class="make-user admin-role remove-admin-role">${_("Remove Staff Access")}</span></a>
+              </li>
+            % else:
+              <li class="action action-role">
+                <a href="#" class="make-staff admin-role add-admin-role">${_("Add Staff Access")}</span></a>
+              </li>
+            % endif
+            <li class="action action-delete ${"is-disabled" if request.user.id == user.id and is_instructor and len(instructors) == 1 else ""}">
+                <a href="#" class="delete remove-user action-icon" data-tooltip="${_("Remove this user")}"><i class="icon-trash"></i><span class="sr">${_("Delete the user, {username}").format(username=user.username)}</span></a>
+            </li>
+          </ul>
+          % elif request.user.id == user.id:
+          <ul class="item-actions user-actions">
+            <li class="action action-delete">
+                <a href="#" class="delete remove-user action-icon" data-tooltip="${_("Remove me")}"><i class="icon-trash"></i><span class="sr">${_("Remove me from this library")}</span></a>
+            </li>
+          </ul>
+          % endif
+
+        </li>
+        % endfor
+      </ol>
+
+      % if allow_actions and len(all_users) == 1:
+      <div class="notice notice-incontext notice-create has-actions">
+        <div class="msg">
+          <h3 class="title">${_('Add More Users to This Library')}</h3>
+          <div class="copy">
+            <p>${_('Adding team members makes content authoring collaborative. Users must be signed up for Studio and have an active account. ')}</p>
+          </div>
+        </div>
+
+        <ul class="list-actions">
+          <li class="action-item">
+            <a href="#" class="action action-primary button new-button create-user-button"><i class="icon-plus icon-inline"></i> ${_('Add a New User')}</a>
+          </li>
+        </ul>
+      </div>
+      %endif
+    </article>
+
+    <aside class="content-supplementary" role="complimentary">
+      <div class="bit">
+        <h3 class="title-3">${_("Library Access Roles")}</h3>
+        <p>${_("Library staff are content co-authors. They have full writing and editing privileges on all content in the library.")}</p>
+        <p>${_("Admins are library team members who can also add and remove other team members.")}</p>
+        <p>${_("Library users cannot edit content in the library, but can view the content and are able to reference or use library elements in their own courses.")}</p>
+      </div>
+
+      <div class="bit">
+        <h3 class="title-3">${_("Organization-Wide Access")}</h3>
```

Comment:
Thanks @bradenmacdonald for the explanation. Just to confirm, you are saying these permissions are at the organization level, so this is different from course team management in Studio > Settings > Course Team?
Q: Do permissions at this organization level translate to having the equivalent level for a library? 
I'm thinking that unless I can explain better, it would be preferable to omit this description in the UI for now, as it probably raises more question than it answers. I can address organizational permissions in the docs rather than here. @marcotuts what do you think?